README.md

```diff
@@ -1,22 +1,55 @@
+# Nix
+
 [![Open Collective supporters](https://opencollective.com/nixos/tiers/supporter/badge.svg?label=Supporters&color=brightgreen)](https://opencollective.com/nixos)
 [![Test](https://github.com/NixOS/nix/workflows/Test/badge.svg)](https://github.com/NixOS/nix/actions)
 
-Nix, the purely functional package manager
-------------------------------------------
+Nix is a powerful package manager for Linux and other Unix systems that makes package
+management reliable and reproducible. Please refer to the [Nix manual](https://nixos.org/nix/manual)
+for more details.
+
+## Installation
+
+On Linux and macOS the easiest way to Install Nix is to run the following shell command
+(as a user other than root):
+
+```
+$ curl -L https://nixos.org/nix/install | sh
+```
+
+Information on additional installation methods is available on the [Nix download page](https://nixos.org/download.html).
+
+## Building And Developing
+
+### Building Nix
+
+You can build Nix via Nix via one of the targets provided by [release.nix](./release.nix):
+
+```
+$ nix-build ./release.nix -A build.aarch64-linux
+$ nix-build ./release.nix -A build.x86_64-darwin
+$ nix-build ./release.nix -A build.i686-linux
+$ nix-build ./release.nix -A build.x86_64-linux
+```
+
+### Development Environment
 
-Nix is a new take on package management that is fairly unique. Because of its
-purity aspects, a lot of issues found in traditional package managers don't
-appear with Nix.
+You can use the provided `shell.nix` to easily bootstrap working development environment:
 
-To find out more about the tool, usage and installation instructions, please
-read the manual, which is available on the Nix website at
-<https://nixos.org/nix/manual>.
+```
+$ nix-shell
+$ ./bootstrap.sh
+$ ./configure
+$ make
+```
 
-## Contributing
+## Additional Resources
 
-Take a look at the [Hacking Section](https://nixos.org/nix/manual/#chap-hacking)
-of the manual. It helps you to get started with building Nix from source.
+- [Nix manual](https://nixos.org/nix/manual)
+- [Nix jobs on nixos.hydra.org](https://hydra.nixos.org/project/nix)
+- [Nix - A One Pager](https://github.com/tazjin/nix-1p)
+- [NixOS Discourse](https://discourse.nixos.org/)
+- #nix / #nixos on irc.freenode.net
```

Comment:
`#nix: Cannot join channel (+i) - you must be invited`

Is this correct?

---

If nobody has a problem with its presence in the first place, I, personally, would put the third-party `nix-1p` repo at the bottom, to somewhat separate it from the other official resources.

Reply:
> If nobody has a problem with its presence in the first place, I, personally, would put the third-party nix-1p repo at the bottom, to somewhat separate it from the other official resources.

If @edolstra "blesses" the document as useful maybe we can have it here. It's certainly helpful.